lib/Elevate/Components/Acronis.pm

```diff
@@ -0,0 +1,58 @@
+package Elevate::Components::Acronis;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Components::Acronis
+
+=head2 check
+
+noop
+
+=head2 pre_distro_upgrade
+
+Find out:
+Is the Acronis agent installed?
```

Comment:
"Find out:" followed by a question seems rhetorical where we want technical.

"Installation Status of Acronis"

Reply:
I'll go ahead and make that change.